core/vm/jump_table.go

```diff
@@ -86,6 +90,8 @@ func newShanghaiInstructionSet() JumpTable {
 	instructionSet := newLondonInstructionSet()
 	enable3855(&instructionSet) // PUSH0 instruction https://eips.ethereum.org/EIPS/eip-3855
 	enable3860(&instructionSet) // Limit and meter initcode https://eips.ethereum.org/EIPS/eip-3860
+	enable4200(&instructionSet) // Static relative jumps https://eips.ethereum.org/EIPS/eip-4200
+	enable4750(&instructionSet) // Functions https://eips.ethereum.org/EIPS/eip-4750
```

Comment:
Check and activate the appropriate EIPs for Shanghai

Reply:
- [x] 3855 is included in Shanghai, but not for EOF
- [x] 3860 is included in Shanghai and necessary for EOF, but not exclusive to EOF

---

EOF-specific EIPs (are any missing?)

- [ ] 3540 (EVM Object Format v1) :heavy_check_mark: Turns on format validation, but has no modifications for the instruction set.
- [ ] 3670 (EOF - Code Validation) :heavy_check_mark: Implemented, no need for mod at the instruction set.
- [x] 4200 (EOF - Static Jumps) :heavy_check_mark: 
- [x] 4750 (EOF - Functions) :heavy_check_mark:  
- [ ] 5450 (Stack Validation) :question: Implementation? Depends on 4750 (EOF - Functions).

Reply:
EIP-5450 is on the works here https://github.com/lightclient/go-ethereum/tree/eof-5450